Plugins/ActSVG/include/Acts/Plugins/ActSVG/LayerSvgConverter.hpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
```

Comment:
Here and in other files we have the wrong year (5x2021, 2x2020), I think.

Reply:
Oh, thanks! will check.